redpanda/Chart.yaml

```diff
@@ -16,9 +16,13 @@
 apiVersion: v2
 name: redpanda
 description: Redpanda is the real-time engine for modern apps.
+maintainers:
+  - name: redpanda-data
+    url: https://github.com/orgs/redpanda-data/people
 type: application
 version: 1.0.7
-deprecated: true
-appVersion: 21.11.11
+icon: https://redpanda.com/static/skate-panda-9c2e4bc5a1ed0052554e658519a50882.svg
```

Comment:
Should this link be to another resource? I'm worried the SVG filename is generated by a release process (or something) and will no longer work at some point.